sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2264,6 +2264,12 @@ object SQLConf {
     .booleanConf
     .createWithDefault(false)
 
+  val TRANSFORMATION_EXIT_TIMEOUT = buildConf("spark.sql.transformation.exit.timeout")
```

Comment:
let's include the unit the the name, like `....timeoutInSeconds`

Reply:
Yes, it is a better name.